src/map-interactions.react.js

```diff
@@ -325,7 +326,9 @@ export default class Interactions extends Component {
   render() {
     return (
       <div
-        ref="container"
+        ref={container => {
```

Comment:
So if I understand, the proper way to circumvent #310 is create a bound class method for the ref callback like described in https://facebook.github.io/react/docs/refs-and-the-dom.html#caveats

If I change it to that, would that work for you?